website/docs/docs/deploy/ci-jobs.md

```diff
@@ -11,12 +11,12 @@ You can set up [continuous integration](/docs/deploy/continuous-integration) (CI
 
 dbt Labs recommends that you create your CI job in a dedicated dbt Cloud [deployment environment](/docs/deploy/deploy-environments#create-a-deployment-environment) that's connected to a staging database. Having a separate environment dedicated for CI will provide better isolation between your temporary CI schema builds and your production data builds. Additionally, sometimes teams need their CI jobs to be triggered when a PR is made to a branch other than main. If your team maintains a staging branch as part of your release process, having a separate environment will allow you to set a [custom branch](/faqs/environments/custom-branch-settings) and, accordingly, the CI job in that dedicated environment will be triggered only when PRs are made to the specified custom branch. To learn more, refer to [Get started with CI tests](/guides/set-up-ci).
 
+
 ### Prerequisites
 - You have a dbt Cloud account.
 - For the [Concurrent CI checks](/docs/deploy/continuous-integration#concurrent-ci-checks) and [Smart cancellation of stale builds](/docs/deploy/continuous-integration#smart-cancellation) features, your dbt Cloud account must be on the [Team or Enterprise plan](https://www.getdbt.com/pricing/).
-- You must be connected using dbt Cloud’s native Git integration with [GitHub](/docs/cloud/git/connect-github), [GitLab](/docs/cloud/git/connect-gitlab), or [Azure DevOps](/docs/cloud/git/connect-azure-devops).
-    - With GitLab, you need a paid or self-hosted account which includes support for GitLab webhooks and [project access tokens](https://docs.gitlab.com/ee/user/project/settings/project_access_tokens.html). With GitLab Free, merge requests will invoke CI jobs but CI status updates (success or failure of the job) will not be reported back to GitLab. 
-    - If you previously configured your dbt project by providing a generic git URL that clones using SSH, you must reconfigure the project to connect through dbt Cloud's native integration.
+- If you're using [GitLab](/docs/cloud/git/connect-gitlab), you need a paid or self-hosted account which includes support for GitLab webhooks and [project access tokens](https://docs.gitlab.com/ee/user/project/settings/project_access_tokens.html). With GitLab Free, merge requests will invoke CI jobs but CI status updates (success or failure of the job) will not be reported back to GitLab. 
+- If you previously configured your dbt project by providing a generic git URL that clones using SSH, you must reconfigure the project to connect through dbt Cloud's native integration.
```

Comment:
we could either remove this bullet point, or specify that it's only necessary if the user wants full CI benefits that come with the native integrations (like automatic dropping of temporary schemas). since that's already covered [here](https://docs-getdbt-com-git-ly-docs-ci-jobs-dbt-labs.vercel.app/docs/deploy/ci-jobs#trigger-a-ci-job-with-the-api), i'm in favor of dropping it. what do you think?

Reply:
i'm a little hesitant to drop this line since iirc it came from an faq. lemme reach out to support and see if they have more user context about this

Reply:
via git blame, i found the original faq/troubleshooting content where this line came from: 

![Screenshot 2024-01-04 at 5 00 42 PM](https://github.com/dbt-labs/docs.getdbt.com/assets/107218380/6851f646-5258-4bf1-ab43-12500373846e)

if this ^ isn't the case anymore, i'm good with removing the bullet as you suggested.

Reply:
oh, the original content for this is still in the Troubleshooting section. it's the second item on the list from [here](https://docs.getdbt.com/docs/deploy/ci-jobs#troubleshooting). 

it tells people to reconnect via a native integration to `enable dbt Cloud CI` . if this isn't the case anymore, i'm good with removing the bullet as you suggested.

as a follow-up, we might want to remove it from the Troubleshooting section as well.

Reply:
gotcha, this makes sense! since CI & most of its benefits are available to non-native integrations as well now, this isn't accurate anymore (a native integration is not _not_ required to enable dbt Cloud CI). i am in favor of removing both the bullet point & the section from troubleshooting, since they are a little bit misleading now